src/scripts/services/group-service.ts

```diff
@@ -143,7 +154,9 @@ export class OktaGroupService {
         );
       },
       (error: unknown) =>
-        `Failed to list groups because of [${JSON.stringify(error)}].`
+        new Error('Failed to list groups.', {
```

Comment:
**Question:** why are all these changes to `Error` necessary? I think `Error` is better than `string` but I would rather make the change in a separate ticket.

Reply:
Because in the QA statement, it states "verify that every other oktagon operation will fail with the same errors." so i thought i had to normalise all the operations.

Also when left is a string and your remove the last line of the PK, you get this error 
![image](https://github.com/agiledigital-labs/oktagon/assets/124639768/84a25ea1-2d54-471c-89b1-d166b4339c75)

Reply:
I'll remember to next time